Project_1/Styles.css

```diff
@@ -1,76 +1,150 @@
-img {
-    src: "Octopus.jpg";
-    max-width: 200%;
-    height: 700px;
-    width: 110%;
-    float: left;
+/* ==========
+GENERAL
+=============*/
+
+*:before,
+*:after {
+    Box-sizing: Border-box;
 }
 
 body {
     background-color: whitesmoke;
 }
 
+img {
+    max-width: 100%;
+}
+
+/* ===========
+TEXT STYLES 
+==============*/
+
+header {
+    border-bottom: 10px solid #018a80;
+    border-top: 10px solid #018a80;
+    margin: 50px 0 100px;
+}
+
 h1 {
-    font-size: 100px;
-    color: #d07f7d;
-    margin-right: 300px;
-    margin-left: 650px;
-    line-height: 1.3;
+    color: #a72323;
+    font-family: 'Noto Sans', sans-serif;
+    font-size: 95px;
+    margin: 10px 300px 50px 650px;
 }
 
-h2 {
-    font-size: 25px;
+h2,
+h3,
+h4 {
     color: #808080;
-    margin-right: 300px;
-    margin-left: 650px;
-    opacity: 70%;
+    font-family: 'Noto Sans', sans-serif;
     font-style: normal;
     font-weight: normal;
 }
 
-h3 {
-    margin-right: 300px;
-    margin-left: 650px;
-    color: #808080;
-    font-size: 16px;
-    font-weight: normal;
+h2 {
+    font-size: 25px;
 }
 
+h3,
 h4 {
-    margin-right: 300px;
-    margin-left: 650px;
     font-size: 16px;
-    color: #808080;
-    font-weight: normal;
+}
+
+h2,
+h3,
+h4 {
+    margin: 20px 300px 20px 650px;
 }
 
 p {
+    font-family: 'Merriweather', serif;
     font-size: 18px;
-    margin-right: 300px;
-    margin-left: 650px;
     line-height: 2.2;
+    margin: 30px 300px 30px 650px;
 }
 
-/*i am not sure how to edit either image individually, so right now the Author photo is very much warped /*
-/*img {
-    margin-right: 400px;
-    margin-left: 750px;
-    max-width: 100%;
-    height: 200px;
-    width: 50%;
-    src: "Carl_Zimmer.jpg";/*
+figcaption {
+    background-color: whitesmoke;
+    border: 3px solid whitesmoke;
+    font-size: 13px;
+    margin-top: -50px;
+    position: absolute;
+}
+
+/*===========
+IMAGES
+=============*/
+
+.octopus-image {
+    margin: 70px 200px 50px 500px;
+    width: 110%;
+}
+
+.author-photo {
+    background-color: #018a80;
+    border: 10px solid #018a80;
+    margin: 150px 350px 100px 650px;
+    width: 9%;
+}
+
+/* ============
+CLASSES
+===============*/
+
+.quote2,
```

Comment:
Numbering these classnames is okay, but maybe it would be clearer to name them based on their function, e.g.
quote-open
quote-close
?